getFeatureFlagById/main.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/Real-Dev-Squad/feature-flag-backend/database"
+	"github.com/Real-Dev-Squad/feature-flag-backend/utils"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"log"
+	"net/http"
+)
+
+func handler(req events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
+	id, found := req.PathParameters["flagId"]
+
+	
+	featureFlag, err := database.ProcessGetFeatureFlagByHashKey(utils.Id, id)
+	if err != nil {
+		return utils.ServerError(err)
+	}
+
+	if featureFlag == nil {
+		log.Println("Feature Flag not found")
+		return utils.ClientError(http.StatusNotFound, "Feature flag not found")
+		
+	}
+	log.Println(featureFlag, " is the feature flag")
+	
+	//marshal the item to JSON
+	jsonResponse, err := json.Marshal(featureFlag)
+	if err != nil {
+		log.Println("Error converting FeatureFlag to JSON")
+		return utils.ServerError(err)
+	}
+
+	response := events.APIGatewayProxyResponse{
+		StatusCode: http.StatusOK,
+		Headers: map[string]string{
+			"Content-Type": "application/json",
+		},
+		Body: string(jsonResponse),
```

Comment:
do we need to pass everything that we get from the DB? it is exposing your data model as is. if not everything is required maybe it is a good idea to have another struct and only send back what is required.

Reply:
Sure, will make the changes and raise the PR

Reply:
Made changes here in this line https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32/commits/36f697e413868af73c75dd5a71ceaf5a66666f36#diff-8a93b4ae6fdbffe07fb8d1afb9805a3ca03417060e0c47368b0e6c2299f4984cR98
@dev-gv